medusa/databases/__init__.py

```diff
@@ -1,20 +1,3 @@
 # coding=utf-8
 
```

Comment:
I think that if we're removing the headers. We should at least add the module docstrings. Whay you guys think?